packages/shared/src/lib/log.ts

```diff
@@ -196,6 +196,7 @@ export enum TargetType {
   StreaksMilestone = 'streaks milestone',
   PromotionCard = 'promotion_card',
   MarketingCtaPopover = 'promotion_popover',
+  MarketingCtaPopoverSmall = 'promotion_popover_small',
```

Comment:
@idoshamun @rebelchris created new log type for the new CTA variant.

Reply:
great!

Reply:
@idoshamun  could you add it to the gdocs file, if it needs to be added there?

Reply:
Added

Reply:
Thanks!